test/compile/common.test.ts

```diff
@@ -129,4 +129,12 @@ describe('Common', () => {
       expect(mergeTitle('title1', 'title2')).toBe('title1, title2');
     });
   });
+
+  describe('formatSignalRef()', () => {
+    it('should format headers that are ordinal', () => {
```

Comment:
(For the test description) is this only header?  seems like it's for any general field def.

Reply:
Good point. I will update the description.